<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="lblOld">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>580</width>
     <height>401</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblNew">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>580</width>
     <height>401</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnText">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>630</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>630</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存图片</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>430</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxImgDealChoice">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>480</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>对比度</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>亮度</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>滤波</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>二值化</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>二值化（反）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>边缘检测canny</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>边缘检测laplacian</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>边缘检测sobel</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>500</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="title">
    <string>滤波</string>
   </property>
   <widget class="QRadioButton" name="rBtnBlur">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Blur</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBtnGaussian">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaussian</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBtnMedian">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Median</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBtnBilateral">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bilateral</string>
    </property>
   </widget>
  </widget>
  <widget class="QSlider" name="hsrChangeSize">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>550</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSlider" name="hsrRevolve">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>580</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>720</number>
   </property>
   <property name="value">
    <number>360</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblOldHeight">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblOldWidth">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblChangeSize">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>550</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>缩放</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRevolve">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>580</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>旋转</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="letNewHeight">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>480</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="letNewWidth">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>510</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="rBtnOriginalProportion">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>440</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保持比例</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnImg2Base">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>430</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>输出图片转base64</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBase2Img">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>430</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>base64转图片</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxImgTypeChoice">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>430</y>
     <width>82</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>png</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>jpg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ico</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>jpeg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bmp</string>
    </property>
   </item>
  </widget>
  <widget class="QTextBrowser" name="tbrBase64">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>470</y>
     <width>381</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
